src/Microsoft.AspNet.WebUtilities/BufferedReadStream.cs

```diff
@@ -289,7 +289,7 @@ public bool EnsureBuffered(int minCount)
         {
             if (minCount > _buffer.Length)
             {
-                throw new ArgumentOutOfRangeException(nameof(minCount), minCount, "The value must be smaller than the buffer size: " + _buffer.Length);
+                throw new ArgumentOutOfRangeException(nameof(minCount), minCount, $"The value must be smaller than the buffer size: {_buffer.Length.ToString()}");
```

Comment:
Note we do not use string interpolation for exception messages because they will eventually be moved to resource strings instead.